app/controllers/sites_controller.rb

```diff
@@ -287,7 +287,16 @@ def tracking_method
 
   private
     def site_params
-      params.require(:site).permit(:palette, :layout, :show_facets, :alternative_title, :search_type, :image_uris, image_uris: [])
-      .tap { |p| p['image_uris']&.delete_if { |v| v.blank? } }
+      params.require(:site).permit(:palette, :layout, :show_facets, :alternative_title, :search_type, :image_uris, :editor_uids, image_uris: [])
+      .tap do |p|
+        p['image_uris']&.delete_if { |v| v.blank? }
+        if can?(:admin, @subsite)
+          p['editor_uids']&.strip!
+          p['editor_uids'] = p['editor_uids'].split(/[\s,]+/)
+        else
+          p['editor_uids'] = @subsite.editor_uids
+        end
+        p
```

Comment:
```suggestion
```
No need for `p` at the end since `tap` always returns the object it was called on.